deploy.sh

```diff
@@ -1,8 +1,18 @@
 #!/bin/bash
 
-PROFILE="stell"
+# The following variables must be personalized before script execution
+PROFILE="stell-gold"
```

Comment:
This workshop is intended for customers to use as well. Having stelligent specific vars will get confusing